### PR TITLE
Ancient

### PR DESCRIPTION
**Pull request by alfonsotech**

### CODE REVIEW

File: components/Ancient.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+       })
+    .then( response => {
+      this.setState({
+        courses: response
+      })
+    })
+  }
+
+  render() {
+
```
Comment (pllearns) [inline components/Ancient.js]: I would minimize the whitespace between the render and conditional

File: components/Ancient.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    } else {
+      return (<div>
+        {this.state.courses.map((data, index) => (
+
+          <article>
+            <ul>
+              <form className='checkbox-form'>
+                <input type='checkbox' className='checkbox' />
+              </form>
+              <a href="{data.courseurl}" target="_blank"><li className="title">{data.title}</li></a>
```
Comment (pllearns) [inline components/Ancient.js]: This is different than the prework issue?

File: components/Prework.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      return response.json()
+       })
+    .then( response => {
+      this.setState({
+        courses: response
+      })
+    })
+  }
+
+
```
Comment (pllearns) [inline components/Prework.js]: Way too much whitespace here. Could we close this a little?

**Comment by alfonsotech** [comment]
The issue with data.courseurl was that I had originally set the prop in the database in camelCase, but when I looked at what was being sent to the endpoint, it was being forced to lowercase, so when I was trying to retrieve the data.courseUrl, that wasn't working. I had a similar issue with course.mediatype versus course.mediaType. It now works for all the endpoints. Stupid formatting issue, and I'm still not sure why it was forced into lowercase.

**Pull request merged**